projects/storefrontapp-e2e-cypress/cypress/integration/SetUp/setup-for-persistent-user.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { config, login, setSessionData } from '../../support/utils/login';
```

Comment:
Since "setup-for-persistent-user.spec.ts" only creates the persistent user when it's not already present, we might consider simply adding this logic to checkout-as-persistent-user.ts.  This way we eliminate a manual configuration step to in each backend where we want to run the tests from. (plus each time we initialize all of those those)

Reply:
The reason why it's in a separate file is that we want to make sure the user already exists before running the test. This test is to specifically make sure that an existing user today, will still be able to login and checkout with his saved info if we upgrade the system tomorrow. Thus, the user should be created before the system is upgraded. 
Ideally, dev17 wouldn't be initialized often. So we wouldn't need to create the user often.

Reply:
After discussing with Patrick, since we never upgrade our backend, we won't really be testing the persistent user scenario. So, I'll move the "create specific user" part to the test, this way we'll test creating/logging in a specific user, and checkout with saved address & payment info.